_posts/2021-03-09-gsoc2021.md

```diff
@@ -0,0 +1,118 @@
+| 5       | [Extend MDAnalysis Interoperability]({{ site.github.wiki }}/Project-Ideas-2021#project-5-extend-interoperability)                  | medium     | Extend converters module to other relevant packages                                            | Python                      | @lilyminium, @IAlibay  |
+| 6       | [General unit cell representation]({{ site.github.wiki }}/Project-Ideas-2021#project-6-general-unit-cell-representation)                  | easy     |  change unit cell representation to keep track of box rotations and so improve analysis of simulations under periodic boundary conditions | Python                      | @orbeckst  |
+
+# Information for Students
+
+You must meet our [own requirements] if you want to be a student with MDAnalysis
+this year (read all the docs behind these links!). You must also meet the
+[eligibility criteria]. Our [GSoC FAQ][] collects common questions
+from students.
+
+The MDAnalysis community values diversity and is committed to
+providing a productive, harassment-free to every member. Our [Code of
+Conduct] explains the values that we as a community uphold. *Every
+member (and every GSoC student) agrees to follow the [Code of
+Conduct]*.
+
+As a start to get familiar with MDAnalysis and open source development you
+should follow these steps:
+
+## Complete the Quick Start Guide
+
+We have a [Quick Start Guide] explaining the basics of MDAnalysis. You
+should go through it at least once to understand how MDAnalysis is
+used. Continue reading the [User Guide] to learn more.
+
+## Introduce yourself to us
+
+Introduce yourself on the [mailing list]. Tell us what you plan to work
+on during the summer or what you have already done with MDAnalysis.
+
+## Close an issue of MDAnalysis
+
+You must have *at least one commit in the development branch of
+MDAnalysis* in order to be eligible, i.e. you must demonstrate that
+you have been seriously engaged with the MDAnalysis project.
+We have a list of [easy bugs][] and suggested [GSOC Starter
+issues][GSOC Starter] to work on in our issue tracker on GitHub. We
+also appreciate if you write more tests or update/improve our
+documentation.
+
+We recommend you start your application by working on an issue. It will give
+you a better understanding of MDAnalysis as a project and improve the quality
+of your application.
+
+To start developing for MDAnalysis have a look at our
+guide on [contributing to MDAnalysis][dev-guide] and write to us on
+the [mailing list][] if you have more questions about setting up a
+development environment or how to contribute.
+
+
+— @IAlibay, @orbeckst, @richardjgowers, @fiona-naughton,
+@micaela-matta, @hmacdope, @lilyminium
```

Comment:
I believe Micaela can't do it this year -- sorry (I copied her over from last year's page).